go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -69,6 +69,11 @@ var sequenceFields = []*querypb.Field{
 	},
 }
 
+func (qre *QueryExecutor) addStats(planName string, queryCount int64, duration, mysqlTime time.Duration, rowCount, errorCount int64) {
+	qre.tsv.qe.AddStats(planName, qre.plan.TableName().String(), queryCount, duration, mysqlTime, rowCount, errorCount)
```

Comment:
We need some handling for the case where the table name is unknown...

For consistency it should use "Join" as the "table name" in that case.

Reply:
Hmm. Where there's an unknown table identifier, it looks like qre.plan.TableName().String() is just the empty string which I think will be fine for our metrics. Would you prefer "Join" over that?